.github/workflows/android.yml

```diff
@@ -22,15 +22,15 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build-demo-android:
-    name: build-demo-android
+  build-llm-demo:
+    name: build-llm-demo
     uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
     strategy:
       matrix:
-          tiktoken: [OFF, ON]
+          android-abi: [arm64-v8a, x86_64]
```

Comment:
Instead of triggering 4 jobs and making 8 aars (tokenizer x abi x so), we should probably make arm64-v8a and x86_64 in the same job and same AAR. Note that AAR itself should contain different ABIs so that only we need only one AAR for different devices or emulators. It's also Android suggested layout in https://developer.android.com/studio/projects/android-library#aar-contents

Reply:
Good point that it will make deployment to device and emulator much easier.

Reply:
Addressed